fdbserver/storageserver.actor.cpp

```diff
@@ -7930,12 +7950,12 @@ void ShardInfo::addMutation(Version version,
                             MutationRefAndCipherKeys const& encryptedMutation) {
 	ASSERT((void*)this);
 	ASSERT(keys.contains(mutation.param1));
-	if (adding)
+	if (adding) {
```

Comment:
Thanks for adding `{ }` even for single line statement, imo it improves readability and avoids accidental bugs in future :)